### DISCUSSION TITLE
How to set MTU size for Transmission?

### DISCUSSION THREAD

**Discussion by yutayu** [transmission/transmission#2523] [Q&A]
How?
I want to set MTU size for transmission  lower than ethernet.

**Comment by lighterowl** [comment]
Not sure if this is something that a torrent client should be doing, since I fail to see the link between torrents and MTU (maybe if you're using jumbo frames, but then again, not directly related to torrents). Besides, these settings are usually set per-interface, so including this option would potentially affect all other applications currently using it.
What would the rationale be behind including an option like this? Why can't you use your OS's tools to do it, or - even better - include it in your Transmission's startup script?

**Comment by lvella** [comment]
This begs an important question: does Transmission performs path MTU discovery when using UDP?

**Comment by yutayu** [comment]
I have forgotten to post this thread lol
@xavery 
I thought if MTU is settled as OS function,whole program doesn't fit MTU different from interface.
